projects/core/types/direction.ts

```diff
@@ -2,6 +2,12 @@ export type TuiHorizontalDirection = 'left' | 'right';
 
 export type TuiVerticalDirection = 'top' | 'bottom';
 
-export type TuiSideDirection = 'bottom-left' | 'bottom-right' | 'top-left' | 'top-right';
+export type TuiSideDirection =
```

Comment:
I think this is incorrect behavior, since the style changes affected only the hint component